Backend-Engineer.md

```diff
@@ -21,25 +21,26 @@ You will build and maintain production-level services. We mostly use **Golang**.
   - Pub/Sub
   - Deployment to staging and production environment
 - Test-Driven Development: write unit and integration tests
-- Scrum
+- Scrum routines
   - Sprint planning
   - Daily standup
+  - Sprint Review
   - Retrospective
 
 ## Qualifications
 
 - Kurio's values (Get sh*t done, Be Bold, Stay Curious, Excellence, Collaboration)
-- Strong knowledge on computer science
+- Deep knowledge on computer science (things like data structure, OOP)
 - Golang
 - Understand and able to build a RESTful API
 - Able to work with SQL and NoSQL like MySQL and MongoDB (DynamoDB, Spanner, etc. is a plus)
-- Familiar with Git and its workflow
+- Familiar with Git workflow and its principles
 
 ## Preferred qualifications
 
-- Experienced in Test-Driven Development
-- Familiar with NodeJS
+- Test-Driven Development (TDD)
+- Familiar with at least one general-purpose language (i.e. Python, Javascript)
```

Comment:
JavaScript is more commonly used to address the language that runs on the browser, maybe we should stick with Node.js

```suggestion
- Familiar with at least one general-purpose language (i.e. Python, Node.js)
```